src/forms/FormLines.stories.tsx

```diff
@@ -10,34 +12,114 @@ export default {
 export function FlatList() {
   return (
     <FormLines>
-      <TextField label={"First"} value={"first"} onChange={() => {}} />
-      <TextField label={"Last"} value={"last"} onChange={() => {}} />
+      <TextField label="First" value="first" onChange={() => {}} />
+      <TextField label="Last" value="last" onChange={() => {}} />
     </FormLines>
   );
 }
 
 export function SmallFlatList() {
   return (
     <FormLines width="sm">
-      <TextField label={"First"} value={"first"} onChange={() => {}} />
-      <TextField label={"Last"} value={"last"} onChange={() => {}} />
+      <TextField label="First" value="first" onChange={() => {}} />
+      <TextField label="Last" value="last" onChange={() => {}} />
     </FormLines>
   );
 }
 
-export function SideBySide() {
+export function SideBySideMedium() {
   return (
     <FormLines>
       <FieldGroup>
-        <TextField label={"First"} value={"first"} onChange={() => {}} />
-        <TextField label={"Middle"} value={"middle"} onChange={() => {}} />
+        <TextField label="First" value="first" onChange={() => {}} />
+        <TextField label="Middle" value="middle" onChange={() => {}} />
       </FieldGroup>
       <TextField label="Address" value="123 Main" onChange={() => {}} />
-      {/* Having three is too wide, so not supported.*/}
       <FieldGroup>
-        <TextField label={"First"} value={"first"} onChange={() => {}} />
-        <TextField label={"Middle"} value={"middle"} onChange={() => {}} />
-        <TextField label={"Last"} value={"last"} onChange={() => {}} />
+        <Switch label="Primary" labelStyle="form" selected={false} onChange={() => {}} />
+        <Switch label="Signatory" labelStyle="form" selected={true} onChange={() => {}} />
+      </FieldGroup>
+      <FieldGroup>
+        <TextField label="Title" value="Engineer" onChange={() => {}} />
+        <span />
+      </FieldGroup>
+      <FieldGroup>
+        <TextField label="First" value="first" onChange={() => {}} />
+        <TextField label="Middle" value="middle" onChange={() => {}} />
+        <TextField label="Last" value="last" onChange={() => {}} />
+      </FieldGroup>
+      <FieldGroup widths={[2, 1, 2]}>
```

Comment:
`FieldGroup` learned a new `widths` which if numbers are `fr` units, but can also be "30px". They're fed into a `gridTemplateColumn`.